public/assets/sass/pages/resources/important-links.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+
+section {
+    max-width: 600px;
+    margin: auto;
+    text-align: center;
+
+    .list {
+        list-style-type: none;
+        padding-left: 0;
+
+        .list-item-content {
+            border: 2px solid var(--mathsoc-pink-2);
+            border-radius: 3em;
+            margin: 1em;
+
+            .list-item-link {
+                text-decoration: none;
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                padding: 1em;
+
+                .list-item-icon {
+                    height: 100%;
+                    width: auto;
+                    padding-right: 1em;
+                }
+                
+                .list-item-title {
+                    margin: 0;
+                    font-weight: 400;
+                    color: var(--mathsoc-pink-2);
+                    padding-left: 1em;
+                }
+            }
+        }
+
+        .list-item-content:hover {
+            background-color: var(--mathsoc-pink-2);
+            color: white;
+            transition: all .25s ease-in;
```

Comment:
If we move these `transition` attribute into the non-hover blocks for these elements, they'll be applied on both ease-in and ease-out! Right now, they only apply on ease-in; once the user stops hovering on the link, it stops applying `:hover` style, which means it loses the smooth transition you've added and reverts to instantaneously changing.

Reply:
Oh I see. Sorry didn't even realize that haha.